site/public/images/icons/sprite.svg

```diff
@@ -55,4 +55,8 @@
+  <svg id="product news" class="svg-icon" viewBox="0 0 36 36" fill="none"><circle cx="18" cy="18" r="18" fill="white"/><circle cx="18" cy="18" r="18" fill="#D9F4D7"/><path d="M22 9H11C9.9 9 9 9.9 9 11V25C9 26.1 9.9 27 11 27H25C26.1 27 27 26.1 27 25V14L22 9ZM25 25H11V11H21V15H25V25ZM13 23H23V21H13V23ZM18 13H13V15H18V13ZM13 19H23V17H13V19Z" fill="#1A7236"/></svg>
```

Comment:
Two comments here:
* Two of these icons appear to embed PNGs. When creating and using icons, we should not embed raster images into our vector images
* Because these are going to change, I'd prefer not to merge these at all for the time being, wait for the updated icons instead, and do a follow-up PR with them

Reply:
While digging around the figma, I've found the original vector paths for these - will swap them in here!